### PR TITLE
Create Header and Welcome Area

### PR DESCRIPTION
**Pull request by codedbystuart**
## Features
* To this point I have setup the linters, github action to run the linters.
* I have also created the Navbar and Welcome Area for the Website.

Screenshot 2022-11-30 at 22 47 08

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    rel="stylesheet">
+  <link rel="stylesheet" href="assets/css/styles.css">
+</head>
+
+<body>
+  <!-- Header -->
+  <header class="header">
+    <ul class="header-nav">
+      <li class="nav-link"><a href="#"><img src="/assets/images/mobile/menu.svg" alt=""></a></li>
+    </ul>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly wrap the mobile menu icon in a `nav` tag instead of `ul` and `li` for better semantics 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    </h1>
+    <p class="intro-description">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+      works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+    <button class="intro-button">
+      Start Collaboration
+    </button>
+  </section>
```
Comment (ha-manel) [inline index.html:40]: Awesome job here 👏 You used all the right tags, keep rocking 🚀

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @codedbystuart 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️
* Your headline looks awesome, looking forward to seeing the rest of your website ✨ 

## Required Changes:
- [ ] Your commit messages are descriptive 👏 It would make them even more awesome if they were capitalized like "Fix linters" 👍 
_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @codedbystuart , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**